src/helpers/helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+
+export const TableNames = Object.freeze({
+  USER: process.env.USER_TABLE_NAME ?? '',
+  URL: process.env.URL_TABLE_NAME ?? '',
+});
+
+export const createResponse = (rawResponse: APIGatewayProxyResult) => ({
+  statusCode: rawResponse.statusCode,
+  body: rawResponse.body,
+  headers: rawResponse.headers ?? {
+    'content-type': 'application/json',
+  },
+});
```

Comment:
here you can use destructuring:

```
const DEFAULT_HEADER = Object.freeze({
    'content-type': 'application/json',
  });


export const createResponse = ({statusCode, body, headers =  DEFAULT_HEADER }: APIGatewayProxyResult) => ({
  statusCode,
  body,
  headers,
});
```

Reply:
I think, i can do something like this

```typescript
export const createResponse = <T>({
  statusCode,
  body,
  headers = DEFAULT_HEADERS,
}: MiddyResult<T>): APIGatewayProxyResult => {
  if (headers['content-type'] === 'application/json') {
    return {
      statusCode,
      body: JSON.stringify(body),
      headers,
    };
  }

  return {
    statusCode,
    body: String(body),
    headers,
  };
};
```

Reply:
Yes, a way better solution than mine, thanks